Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3011,6 +3011,41 @@ internal static bool ShouldDrawWarningIconForBinding(string bindingPath)
 
 #if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
 
+#if UNITY_EDITOR
+        [InitializeOnLoad]
+        public static class PlayStateNotifier
+        {
+            static PlayStateNotifier()
+            {
+                EditorApplication.playModeStateChanged += ModeChanged;
+            }
+
+            static void ModeChanged(PlayModeStateChange playModeState)
+            {
+                // EnteredEditMode  Occurs during the next update of the Editor application if it is in edit mode and was previously in play mode.
+                // ExitingEditMode  Occurs when exiting edit mode, before the Editor is in play mode.
+                // EnteredPlayMode  Occurs during the next update of the Editor application if it is in play mode and was previously in edit mode.
+                // ExitingPlayMode  Occurs when exiting play mode, before the Editor is in edit mode.
+                switch (playModeState)
+                {
+                    case PlayModeStateChange.EnteredEditMode:
+                        actions?.Disable();
+                        break;
+
+                    case PlayModeStateChange.ExitingEditMode:
+                        break;
+
+                    case PlayModeStateChange.EnteredPlayMode:
+                        actions?.Enable();
```

Comment:
I think MonoBehaviour's Start() is called before PlayModeStateChange.EnteredPlayMode
So when I (previously - not with this PR) tested to see if actions were enabled in my Start function, it was not enabled.

But perhaps that doesn't matter. I think we might just need to be clear in the docs when the InputSystem.actions get enabled.


And another comment:
PlayModeStateChange.ExitingPlayMode gets hit before OnDisable 
(But I see you are using EnteredEditMode which is later)